README.md

```diff
@@ -136,42 +136,41 @@ plugin. You can turn on/off an option by adding/removing the corresponding flag
 some meaningful syntax in the source file. This option is enabled by default.
 
 
+
 ## Developer's Guide
 
-To set up your development environment, perform the following steps:
+### Build from source
 1. Install the newest Node.js version for your operating system (OS); see the [Node.Js homepage](https://nodejs.org/en/).
 2. Install a Git client for your OS; see the [Git homepage](https://git-scm.com/downloads).
-3. Install dependencies using `npm install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.)
-   See the [npm Docs](https://docs.npmjs.com/cli/v8/commands/npm-install).
-4. Enable the corepack utility by running `corepack enable`, as superuser.
```

Comment:
It's nice to see that this dependency is no longer needed!

Reply:
Me too.